test/test_utils.py

```diff
@@ -22,24 +22,6 @@ def rec_bin(x):
             return utils.decode_hex(x)
 
 
-# def deploy_solidity_contracts(solc_config_sources, allow_paths, contract_file, contract_name):
```

Comment:
delete this?

Reply:
This PR does delete this code ... I'm confused

Reply:
You're right, sorry